sdks/python/test-suites/tox/py38/build.gradle

```diff
@@ -26,6 +26,10 @@ applyPythonNature()
 // Required to setup a Python 3 virtualenv and task names.
 pythonVersion = '3.8'
 
+def posargs = project.findProperty("posargs") ?: ""
+def runDataframesTests = (posargs == "" || posargs.contains("dataframes"))
```

Comment:
py38/build.gradle has more content than other versions and these additional contents are for precommit running on a single version. We should gather these single version lines together (under comment TODO(BEAM-12000) below)  to make future version bump easier.